core/src/main/java/com/vityuk/ginger/cache/LoadingCache.java

```diff
@@ -0,0 +1,15 @@
+/**
+ * com.vityuk.ginger.cache.LoadingCache.java
+ *
+ * Copyright (c) 2007-2015 UShareSoft SAS, All rights reserved
+ * @author UShareSoft
+ */
+package com.vityuk.ginger.cache;
+
+import java.util.concurrent.ExecutionException;
+
+public interface LoadingCache<T, T1> {
```

Comment:
I would replace T and T1 with something more descriptive, like K for key and V for value.